apps/fulfillment-e2e/src/support/page_fragments/picking-in-progress-modal.fragment.ts

```diff
@@ -1,5 +1,5 @@
 export class PickingInProgressModalFragment {
-  getWrapper = () => cy.get('oryx-picking-in-progress-modal');
+  getWrapper = () => cy.get('oryx-picking-picking-in-progress-modal');
```

Comment:
I believe we can make and exception here, because **picking-picking** looks not very impressive